source/includes/steps-configure-settings.yaml

```diff
@@ -0,0 +1,49 @@
+title: Open the Settings View.
+level: 4
+ref: open-settings
+content: |
+
+    .. list-table::
+       :header-rows: 1
+   
+       * - Operating System
+         - Methods
+
+       * - MacOS
+         - .. list-table::
+              :header-rows: 1
+
+              * - Method
```

Comment:
The nested table makes this difficult to digest with `Methods` and `Method` so close to one another. It's hard to tell what I'm looking at.

I'd suggest structuring this more like the staged copy here: 

<img width="735" alt="Screen Shot 2020-05-11 at 4 49 50 PM" src="https://user-images.githubusercontent.com/29379027/81610397-75005900-93a7-11ea-98fb-4d5bead80780.png">


So, three columns:

Operating System, Method, Actions

To get around the fact that `File` and `Code` differ between Windows and macOS, I'd just say `File / Code` for the `Any` row.

Reply:
Sounds good, the non-nested table looks better. It was just about the File/Code distinction.